src/boot/AppEventHandlers.js

```diff
@@ -95,6 +96,17 @@ class AppEventHandlers extends PureComponent<Props> {
     if (state === 'background' && Platform.OS === 'android') {
       NativeModules.BadgeCountUpdaterModule.setBadgeCount(unreadCount);
     }
+    if (state === 'active' && Platform.OS === 'ios') {
+      // TODO: This API is deprecated in react-native-notifications release
+      // 2.0.6-snapshot.8; see #3647.
+      try {
+        // We don't know the behavior if this is called before
+        // NotificationsIOS.requestPermissions(), so, catch any errors silently.
+        NotificationsIOS.consumeBackgroundQueue();
+      } catch (e) {
+        // Do nothing
+      }
```

Comment:
At the very least we need to feed these errors to Sentry. In particular, if we upgrade `react-native-notifications` to a later version without fixing this call, it *will* start throwing, and we may never notice! (Flow has no functional typing information for this module, so won't complain.)